.github/workflows/CI.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on:
+  push:
+    branches: [ master, develop ]
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
The [default type](https://help.github.com/en/actions/reference/events-that-trigger-workflows#pull-request-event-pull_request) for `pull_request` seems sufficient, so I don't think that adding this is needed.